sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ShouldImplementExportedInterfaces.cs

```diff
@@ -91,4 +91,18 @@ class ExportingMembers_Are_Ignored
         [Export(typeof(Exported))]
         public NotExported MyMethod() { }
     }
+
+    interface ISomething<T> { }
+    public class BaseThing { }
+
+    [Export(typeof(ISomething<BaseThing>))]
+    public class BaseSomethingImplementation : ISomething<BaseThing>

```

Comment:
Other test cases:
* generic export that doesn't implement any interfaces -> non-compliant
* generic export where the export is of type _ISomething[X]_ and the class implements _ISomething[Y]_ -> non-compliant